src/moj/components/header/template.njk

```diff
@@ -14,10 +14,37 @@
+        <svg aria-hidden="true" focusable="false" class="moj-header__logotype-crest-2024" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
+
+        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
we have an `svg` within an `svg` here - you can remove line 19

Reply:
Amended.